packages/react/src/NavList/NavList.test.tsx

```diff
@@ -82,8 +82,13 @@ describe('NavList', () => {
     const groupHeading = getByText('Group heading')
     const groupTitle = getByText('Overview')
 
+    // Find the div wrapping the "Overview" text
+    const groupTitleWrapper = groupTitle.closest('div')
+
+    console.log(window.getComputedStyle(groupTitleWrapper as Element).cssText)
```

Comment:
Looks like this needs to be removed

Reply:
I've decided to just remove this test because we have it covered with VRT. Our jest config can't read CSS from CSS Module files yet and while we should fix that, I'm not sure its a blocker for this PR.